tests/unit/autoprompt/test_output.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import json
+
+from awscli.clidriver import create_clidriver
+from awscli.autocomplete import parser
+from awscli.autoprompt.output import OutputGetter
+
+from awscli.testutils import unittest
+from tests.unit.autocomplete import InMemoryIndex
+
+
+class TestOutputGetter(unittest.TestCase):
+    def setUp(self):
+        index = InMemoryIndex({
+                'command_names': {
+                    '': [('aws', None)],
+                    'aws': [('s3api', 'Amazon Simple Storage Service')],
+                    'aws.s3api': [('get-object', None)],
+                },
+                'arg_names': {
+                    '': {
+                        'aws': ['query', 'output'],
+                    },
+                    'aws.s3api': {
+                        'get-object': [],
+                    },
+                },
+                'arg_data': {
+                    '': {
+                        'aws': {
+                            'query': (
+                                'query', 'string', 'aws', '', None, False,
+                                False),
+                            'output': (
+                                'output', 'string', 'aws', '', None, False,
+                                False),
+                        }
+                    },
+                    'aws.s3api': {
+                        'get-object': {}
+                    }
+                }
+            })
+        self.parser = parser.CLIParser(index)
+        self.driver = create_clidriver()
```

Comment:
We should make sure that for these tests we are using a stubbed environment. It is possible to set in your `~/.aws/config` file the output format. So we do not want to unintentionally be using that when some of these tests assume that the fallback is configured to json.